packages/client/lib/components/Board.tsx

```diff
@@ -25,38 +24,42 @@ const Board = (props: Board) => {
   }, []);
 
   return (
-    <>
-      {props.playerData.board.map((column, i) => (
-        <div key={i} id={`column-${i}`}>
-          {column.map((value, j) => (
-            <div
-              key={j}
-              id={`row-${j}`}
-              onMouseUp={() => {
-                if (!props.isShipSelected || !props.setIsShipSelected) return;
-                placeShip(
-                  props.playerData.board,
-                  props.selectedShipIndex,
-                  i,
```

Comment:
Let's send coord arguments as [x,y] instead of two arguments of numbers. It can be confusing with many arguments as to which comes first.

Reply:
I am not sure if I understand your suggestion 100% but I think I see where it is going. Will try.